book.common/src/main/java/com/happystays/book/common/dto/Supplier.java

```diff
@@ -0,0 +1,15 @@
+package com.happystays.book.common.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Supplier {
+    private int supplierId;
+    private String supplierCode;
+    private String supplierName;
+    private String supplierUri;
+}
```

Comment:
we dont need to publish all supplier credential here. Just id only required. Once we take from kafka we can take rest of credential..Fow now we will hardcode the supplier id .later on we have to dynamically take credentials with that id